cpp/velox/CMakeLists.txt

```diff
@@ -52,6 +47,19 @@ if (NOT DEFINED VELOX_HOME)
   message(STATUS "Set VELOX_HOME to ${VELOX_HOME}")
 endif()
 
+execute_process(
```

Comment:
What is this change for? Could you add some notes here?

Reply:
keep cpp/velox module's compile option same as Velox.